packages/financial-templates-lib/src/logger/Logger.ts

```diff
@@ -52,8 +52,10 @@ export function createNewLogger(
   transportsConfig = {},
   botIdentifier = process.env.BOT_IDENTIFIER || "NO_BOT_ID"
 ): AugmentedLogger {
+  const sessionId = Date.now();
```

Comment:
nit: we might prefer a random integer to using Date.now(), as I think the chance of collision with multiple calls to Date.now() around the same time isn't as close to zero as one might expect.

Thoughts on something like this to generate a random int between 0 and max safe int?
```ts
const sessionId = Math.floor(Math.random() * Number.MAX_SAFE_INTEGER);
```